.gitignore

```diff
@@ -53,3 +53,9 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Vscode stuff
+.vscode/
```

Comment:
Please remove this, I don't want every editor's cruft exhaustively listed in the project's ignore file. Rather avoid adding this file when committing.